### ISSUE TITLE
Should XALT strip off non-numeric characters from job_id?

### ISSUE THREAD

**Issue by treydock** [Fahey-McLay/xalt#28]
It looks like the job_id saved by XALT is partial in a Torque environment where the job ID might be something like `270.pitzer-batch.ten.osc.edu`.  There is a configuration flag in Torque that makes job IDs not have the batch schedule in value but we don't set that for 3 of our 4 systems.  Would it be worth updating XALT to strip non-numeric characters from the jobID for just Torque?  The combination of numeric portion of `PBS_JOBID` plus syshost seems sufficient for uniqueness.

This is what ends up in database:

```
# mysql xalt_pitzer -e 'select job_id,syshost from xalt_run'
+-------------+---------+
| job_id      | syshost |
+-------------+---------+
| 270.pitzer- | pitzer  |
| unknown     | pitzer  |
| unknown     | pitzer  |
+-------------+---------+

```

**Comment by rtmclay** [comment]
I have updated XALT 2.1.1 to do this.  Note that the devel branch has been merged into the master branch.  Please test out 2.1.1 to see if it works for you.  Thanks for reporting this issue!

**Comment by treydock** [comment]
Using 2.1.1 and job_id is now entirely numeric, thanks!

**Comment by treydock** [comment]
@rtmclay Folks here at OSC pointed out stripping all but `[0-9]` will break job Arrays.  In Torque a job array would be something like `123[1].batch-server` so likely need to also allow `[` and `]`.  Opened #31 to address the need to support job arrays from Torque.

**Comment by rtmclay** [comment]
I think that this issue can be closed as #31 has solved this.
